com.unity.render-pipelines.universal/Shaders/Terrain/TerrainLit.shader

```diff
@@ -68,6 +68,8 @@ Shader "Universal Render Pipeline/Terrain/Lit"
             #pragma multi_compile _ _MAIN_LIGHT_SHADOWS _MAIN_LIGHT_SHADOWS_CASCADE _MAIN_LIGHT_SHADOWS_SCREEN
             #pragma multi_compile _ _ADDITIONAL_LIGHTS_VERTEX _ADDITIONAL_LIGHTS
             #pragma multi_compile_fragment _ _ADDITIONAL_LIGHT_SHADOWS
+            #pragma multi_compile_fragment _ _REFLECTION_PROBE_BLENDING
+            #pragma multi_compile_fragment _ _REFLECTION_PROBE_BOX_PROJECTION
```

Comment:
Does per-object data work for terrain patches? I had the idea it doesn't.

Reply:
It works per pixel. But it will not use box projection because of the uneven floor will give unwanted results with the current implementation of box projection